packages/devtools_app/test/network_profiler_test.dart

```diff
@@ -268,32 +261,6 @@ void main() {
       await clearTimeouts(tester);
     });
 
-    // Regression test for https://github.com/flutter/devtools/issues/3286.
-    testWidgetsWithWindowSize('can select by clicking on url', windowSize,
```

Comment:
why was this regression test removed?

Reply:
Good catch !
My bad : I started working on this PR before your PR #3287 was merged (the one adding this regression test) and I didn't notice this when merging...

I need some guidance to properly merge our changes on this file.
In my PR, I kept the old tests (timeline) and created a new group for HttpProfile tests (I didn't want to remove the old ones before asking, but didn't like to have to duplicate test code).
In your PR, you "upgraded" the old tests from Timeline profiler to Http profiler.

I really prefer your way, as Timeline profiling for HTTP requests seems to be obsolete now. 
Can I assume this ? (if so I could throw away my changes on this file)

Reply:
yes it is safe to "upgrade" from timeline to the http profile impl. Feel free to do that here.